### PR TITLE
UI: all imported style is just CSS

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

- removes notion of `js.scss`
- removes sass
- removes notion of expensive/cheap CSS
- exports CSS files similarly to css modules (`.red` -> `styles.red`) for use in a components `style` prop
- any non-imported CSS-in-JS object passed to the `style` prop is still treated as 'cheap' CSS.
- handles expensive/cheap CSS behind the scenes: if `styletron` can handle it, it gets it, if not it's passed to `emotion`
- add specific postcss transforms to provide pasteup/breakpoints and cover basic lost sass functionality

_in another PR, we need to lint against invalid rules properly_

## What is the value of this and can you measure success?

In theory, you can now Just Write CSS.

Hopefully makes applying CSS to components simpler:

```css
/* 
this is supported without needing to import anything.
values are derived from pasteup.js
*/
.logo {
    color: black;
    @media (--from-tablet) and (--until-desktop) {
        color: var(--colour-brand-blue);
    }
    @supports (display: flex) {
        display: flex;
    }
}
```
```jsx
import { logo } from './style.css';

export default () => 
    <div style={logo}>
        black text except between tablet and desktop, when it's blue
        <span style={{ color: 'red' }}>red text</span>
    </div>;
```
becomes:
```html
<style>
.a { color: black }
.b { color: red }
</style>

<style media="(min-width: 46.25em) and (max-width: 61.1875em)">.c { color:#005689 }</style>

<style>
@supports ((display: -webkit-box) or (display: flex)) {
    .css-1x0mb51 {
        display:-webkit-box; display: 
        -ms-flexbox; display: flex;
    }
}
</style>

<div class="a c css-1x0mb51">
    ...
    <span class="b>...</span>
</div>
```

### CODE REVIEW

File: ui/__tools__/ui-css-loader.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+const normaliseKeys = styles =>
+    Object.keys(styles).reduce(
+        (normalised, key) =>
+            Object.assign({}, normalised, {
+                [key.replace(/^\./, '')]: styles[key],
+            }),
+        {}
+    );
+
+// bloomin heck - replace 'float's that have been converted by postcss-js
```
Comment (NataliaLKB) [inline ui/__tools__/ui-css-loader.js]: 😂

File: ui/package.json [hunk 2]
```diff
@@ -21,10 +21,10 @@
         "watch:server": "webpack --config ./__config__/webpack.config.dev.js --env.server --watch"
     },
     "dependencies": {
+        "emotion": "^7.3.2",
```
Comment (gustavpursche) [inline ui/package.json]: This is now a duplicated entry ... https://github.com/guardian/frontend/blob/f1dfc10210f9f2f9360a162ccc322435d5e68169/ui/package.json#L50

**Review by gu-stav — APPROVED** [review]
This is an amazing (!!!) PR. ❤️ everything about it.

File: ui/package.json (hunk 2 above)
Comment (sndrs) [reply ui/package.json]: removed from devdeps but not showing

**Pull request merged**